packages/components/psammead-radio-schedule/src/testHelpers/helper.jsx

```diff
@@ -19,7 +19,7 @@ const getSchedule = (service, withLongSummary) => {
   return stateTypes.map((state, index) => ({
     id: index,
     state,
-    stateLabel: sentenceCase(state),
+    stateLabel: service === 'arabic' ? 'مباشر' : sentenceCase(state),
```

Comment:
LGTM overall. Just something small... for consistency, as below we can define these props (durationLabel/stateLabel) above.

Reply:
Defined `durationLabel` above, but this cannot be done for `stateLabel` as it uses a `state` parameter from mapping

Reply:
makes sense, thanks Olga 💯